etc/delfin/delfin.conf

```diff
@@ -10,7 +10,19 @@ db_backend = sqlalchemy
 [scheduler]
 config_path = /etc/delfin/scheduler_config.json
 
-[KAFKA]
+[KAFKA_EXPORTER]
+enable = False
 kafka_topic_name = "delfin-kafka"
 kafka_ip = 'localhost'
 kafka_port = '9092'
+
+[PROMETHEUS_EXPORTER]
+enable = False
+metric_server_ip = 0.0.0.0
+metric_server_port = 8195
+metrics_cache_file = /var/lib/delfin/delfin_exporter.txt
+
+[PROMETHEUS_ALERT_MANAGER_EXPORTER]
+enable = False
+alert_manger_host = 'localhost'
+alert_manger_port = '9092'
```

Comment:
"manager" spell error (same in above line also)

Reply:
done